### PR TITLE
fix(theme): bring back root classname helper and fix docs

### PR DESCRIPTION
**Pull request by amje**

### CODE REVIEW

**Review by korvin89 — APPROVED** [review]

File: src/components/theme/getRootClassName.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {block} from '../utils/cn';
+import type {CnMods} from '../utils/cn';
+
+import {ROOT_CLASSNAME} from './constants';
+
+const b = block(ROOT_CLASSNAME);
+
+export function getRootClassName(mods: CnMods, className: string) {
```
Comment (ValeraS) [inline src/components/theme/getRootClassName.ts:8]: It seems the first parameter is too general, maybe let's make it more strict?

We are not using this method yourself, isn't it?

Comment (amje) [reply src/components/theme/getRootClassName.ts:8]: You mean instead of object allow only known keys, like `theme`?

Comment (ValeraS) [reply src/components/theme/getRootClassName.ts:8]: Yes

File: src/components/theme/getRootClassName.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {block} from '../utils/cn';
+
+import {ROOT_CLASSNAME} from './constants';
+
+const b = block(ROOT_CLASSNAME);
+
+interface RootMods {
+    theme?: string;
```
Comment (ValeraS) [inline src/components/theme/getRootClassName.ts:8]:
```suggestion
    theme?: RealTheme;
```

File: src/components/theme/getRootClassName.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import {block} from '../utils/cn';
+
+import {ROOT_CLASSNAME} from './constants';
+
+const b = block(ROOT_CLASSNAME);
+
+interface RootMods {
```
Comment (ValeraS) [inline src/components/theme/getRootClassName.ts:7]:
```suggestion
interface RootMods {
    nativeScrollbar?: boolean;
```

File: src/components/theme/getRootClassName.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    theme?: string;
+}
+
+export function getRootClassName({theme}: RootMods = {}, className?: string) {
+    return b({theme}, className);
```
Comment (ValeraS) [inline src/components/theme/getRootClassName.ts:12]:
```suggestion
    return b({theme, 'native-scrollbar': nativeScrollbar}, className);
```

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: src/components/theme/getRootClassName.ts (hunk 3 above)
Comment (amje) [reply src/components/theme/getRootClassName.ts:7]: We're planning to remove scroll customization, #1587

**Pull request merged**